src/Modal.soy

```diff
@@ -17,7 +18,7 @@
 		class="modal{$elementClasses ? ' ' + $elementClasses : ''}"
 		style="display: {$visible ? 'block' : 'none'}">
 		<div
-			class="modal-dialog"
+			class="modal-dialog{$dialogClasses ? ' ' + $dialogClasses : ''}"
```

Comment:
I think we might have to do this for server side rendering,  since the default values of the component won't get passed to the template in that scenario.